scripts/ImportExportTest.sh

```diff
@@ -0,0 +1,302 @@
+                            then
+                                printf "\n"
+                                echo "$nth_input_file"
+                                cat obtained_direct_import_export.error
+                                FAILED=$((FAILED + 1))
+                                return 0
+                            else
+                                for obtained_contract in $(jq '.contracts | keys | .[]' obtained_direct_import_export.json 2> /dev/null)
+                                do
+                                    jq --raw-output ".contracts.${obtained_contract}.\"asm\"" obtained_direct_import_export.json > obtained_direct_import_export.asm
+                                    set +e
+                                    DIFF="$(diff expected.asm obtained_direct_import_export.asm)"
+                                    set -e
+                                    if [ "$DIFF" != "" ]
+                                    then
+                                        if [ "$DIFFVIEW" == "" ]
+                                        then
+                                            echo -e "ERROR: JSONS differ for $1: \n $DIFF \n"
+                                            echo "Expected:"
+                                            cat  expected.asm
+                                            echo "Obtained:"
+                                            cat obtained_direct_import_export.asm
+                                        else
+                                            # Use user supplied diff view binary
+                                            $DIFFVIEW expected.asm obtained_direct_import_export.asm
+                                        fi
+                                        _TESTED=
+                                        FAILED=$((FAILED + 1))
+                                        return 0
+                                    fi
+                                done
+                            fi
+
+                            rm obtained.json
+                            rm -f obtained.error
+                            for type in "${types[@]}"
+                            do
+                                rm "obtained.${type}"
+                            done
+                        fi
+
+                        for type in "${types[@]}"
+                        do
+                            rm "expected.${type}"
+                        done
+                    fi
+                done
+                rm expected.json
+            fi
+            if [ -n "${_TESTED}" ]
+            then
+                TESTED=$((TESTED + 1))
+            fi
+        else
+            echo "unknown import test type. aborting."
+            exit 1
+        fi
+    else
+        UNCOMPILABLE=$((UNCOMPILABLE + 1))
+    fi
+}
+
+WORKINGDIR=$PWD
+NSOURCES=0
+
+# check whether SOLC works.
+if ! $SOLC --version > /dev/null 2>&1
```

Comment:
I wonder why shellcheck does not complain about this not being quoted:
```suggestion
if ! "$SOLC" --version > /dev/null 2>&1
```
Looks like there are many instances of this not being quoted all around the file.